test/integration/util/integration_logger.go

```diff
@@ -0,0 +1,103 @@
+// +build integration
+
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	messageInfoMaxLength = 7
+	timeHeaderLength     = 16
+)
+
+var (
+	IntegrationLog *log.Logger
+	logFile        *os.File
+)
+
+func init() {
+	IntegrationLog = log.New(ioutil.Discard, "", 0)
```

Comment:
I don't think cost is an issue, especially since this is a test harness. It is more about letting other people understand the code. So at the very least one would need a comment. 

That said, we are controlling the calls to the logger, right? We call it via the util.LogMessage function. Why don't you add a nil check for the logger there. If the logger is not configured return directly. For good measurement, you can print a message to the console.